app/src/main/java/com/monkeyteam/chimpagne/viewmodels/EventViewModel.kt

```diff
@@ -243,15 +242,7 @@ class EventViewModel(
    * joinEvent in FindEventViewModel
    * --> Call in DetailEventScreen
    */
-  fun joinEvent(
-      eventId: ChimpagneEventId = _uiState.value.id,
-      onSuccess: () -> Unit = {},
-      onFailure: (Exception) -> Unit = {}
-  ) {
-    joinTheEvent(onSuccess, onFailure)
-  }
-
-  fun joinTheEvent(onSuccess: () -> Unit = {}, onFailure: (Exception) -> Unit = {}) {
+  fun joinEvent(onSuccess: () -> Unit = {}, onFailure: (Exception) -> Unit = {}) {

```

Comment:
thank you, it was really disturbing to have joinEvent and joinTheEvent in the same class for the same thing